app/screens/onboarding/slide.tsx

```diff
@@ -0,0 +1,218 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useEffect, useState} from 'react';
+import {View, useWindowDimensions} from 'react-native';
+import Animated, {Extrapolate, interpolate, useAnimatedStyle, useSharedValue, withTiming} from 'react-native-reanimated';
+
+import {OnboardingItem} from '@typings/screens/onboarding';
+import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
+import {typography} from '@utils/typography';
+
+type Props = {
+    item: OnboardingItem;
+    theme: Theme;
+    scrollX: Animated.SharedValue<number>;
+    index: number;
+};
+
+const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => ({
+    image: {
+        justifyContent: 'center',
+        maxHeight: 180,
+    },
+    title: {
+        height: 100,
+        color: theme.centerChannelColor,
+        textAlign: 'center',
+        paddingHorizontal: 20,
+    },
+    fontTitle: {
+        marginTop: 32,
+        ...typography('Heading', 1000, 'SemiBold'),
+    },
+    fontFirstTitle: {
+        ...typography('Heading', 1200, 'SemiBold'),
+        paddingTop: 40,
+    },
+    widthLastSlide: {
+        paddingHorizontal: 50,
+    },
+    firstSlideInitialPosition: {
+        left: 200,
+        opacity: 0,
+    },
+    description: {
+        textAlign: 'center',
+        paddingHorizontal: 20,
+        height: 80,
+        color: changeOpacity(theme.centerChannelColor, 0.64),
+        ...typography('Body', 200, 'Regular'),
+    },
+    itemContainer: {
+        flex: 1,
+        alignItems: 'center',
+        justifyContent: 'center',
+    },
+}));
+
+const FIRST_SLIDE = 0;
+const LAST_SLIDE = 3;
+const FIRST_LOAD_ELEMENTS_POSITION = 400;
+
+const SlideItem = ({theme, item, scrollX, index}: Props) => {
+    const {width} = useWindowDimensions();
+    const styles = getStyleSheet(theme);
+
+    /**
+     * Code used to animate the first image load
+     */
+    const [firstLoad, setFirstLoad] = useState(true);
+
+    const initialImagePosition = useSharedValue(FIRST_LOAD_ELEMENTS_POSITION);
+    const initialTitlePosition = useSharedValue(FIRST_LOAD_ELEMENTS_POSITION);
+    const initialDescriptionPosition = useSharedValue(FIRST_LOAD_ELEMENTS_POSITION);
+
+    const initialElementsOpacity = useSharedValue(0);
+
+    useEffect(() => {
+        if (index === FIRST_SLIDE) {
+            initialImagePosition.value = withTiming(0, {duration: 600});
+            initialTitlePosition.value = withTiming(0, {duration: 800});
+            initialDescriptionPosition.value = withTiming(0, {duration: 1000});
+
+            initialElementsOpacity.value = withTiming(1, {duration: 1000});
```

Comment:
@matthewbirtch initially the opacity is taking one second to go from 0 to 1, that may be the delay you are seeing

Reply:
In fact the entire animation takes 1 second to complete, seems too long for me... try to have them not longer than 500ms